packages/xo-web/src/common/editable/index.js

```diff
@@ -36,6 +28,7 @@ import {
 
 import styles from './index.css'
 
+const ADDON_BUTTON_STYLE = { lineHeight: '1.4' }
```

Comment:
Add the comment so we keep track of this ugly workaround :)
https://github.com/vatesfr/xen-orchestra/blob/be6cc1263230f540115b6ca2e8356c41b422b0c7/packages/xo-web/src/common/select-objects.js#L56-L60